fire/SFMainMod.F90

```diff
@@ -916,28 +926,52 @@ subroutine  crown_damage ( currentSite )
 
           do while(associated(currentCohort))  
              currentCohort%fraction_crown_burned = 0.0_r8
+             currentCohort%passive_crown_FI      = 0.0_r8  !critical fire intensity for passive crown fire
+             currentCohort%crown_fire_flg        = 0       !flag for passvie crown fire ignition
+             currentCohort%ignite_crown          = 0.0_r8  !ratio of ignition of passive crown fire,EQ 14 Bessie & Johnson 1995
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
-                ! Flames lower than bottom of canopy. 
-                ! c%hite is height of cohort
-                if (currentPatch%SH < (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft))) then 
+                
+                ! height_cbb = clear branch bole height at base of crown (m)
+                ! inst%crown = crown_depth_frac (PFT)
+                crown_depth = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft) 
+                height_cbb  = currentCohort%hite - crown_depth
+
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   currentCohort%passive_crown_FI = (0.01_r8 * height_cbb * SF_val_crown_ignition_energy)**1.5_r8
+
+                   if (currentCohort%passive_crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
+                      
+                      ! Initiation of passive crown fire, EQ 14 Bessie and Johnson 1995
+                      currentCohort%ignite_crown = currentPatch%FI/currentCohort%passive_crown_FI
+                      
+                      if (currentCohort%ignite_crown >= 1.0_r8) then
+                         currentCohort%crown_fire_flg = 1  ! passive crown fire ignited
+                         currentCohort%fraction_crown_burned =  1.0_r8
+                      ! else ! evaluate crown damage based on scorch height
+                      endif ! ignite passive crown fire
+                   ! else no crown fire today
+                   endif ! crown fire intensity
+                ! else ! not crown fire plant
+                endif ! evaluate passive crown fire
+                
+                ! Flames lower than bottom of canopy.
+                ! height_cbb is clear branch bole height or height of bottom of canopy 
+                if (currentPatch%SH <= height_cbb) .and. (currentCohort%crown_fire_flg == 0) then &
```

Comment:
@jkshuman this was fine like this, i.e. without the parentheses in the middle near the `.and.` and without the `&` at the end:
```suggestion
                if (currentPatch%SH <= height_cbb .and. currentCohort%crown_fire_flg == 0) then
```
If you prefer more parentheses, you could do 
```suggestion
                if ((currentPatch%SH <= height_cbb) .and. (currentCohort%crown_fire_flg == 0)) then
```

Reply:
@slevisconsulting Yeah, I was hitting a wall debugging this section, and missed the incorrect &. the parentheses got thrown in there as I was working through things. It's running and compiling now, so maybe I will remove the extra parentheses...

Reply:
Now I realize that we don't even need this `if` in the logic because you initialized fraction_crown_burned = 0.0_r8 to begin with.  So we can simply turn the next `else if` into an `if`.  (Sorry for catching these one-at-a-time...)

Reply:
> [...] It's running and compiling now [...]

That's awesome! I'll wait for a "go ahead" from you, to update my `active_crown_fire` branch. Then I will create a PR to replace the accidental one that needs to close.

Reply:
@slevisconsulting good point. I added some comments for clarity around surface fires, and updated this.